packages/@aws-cdk/custom-resource-handlers/test/aws-s3-deployment/bucket-deployment-handler/test.py

```diff
@@ -576,7 +576,7 @@ def test_fails_when_physical_id_not_present_in_update(self):
             "DestinationBucketName": "<dest-bucket-name>",
         }, expected_status="FAILED")
 
-        self.assertEqual(update_resp['Reason'], "invalid request: request type is 'Update' but 'PhysicalResourceId' is not defined")
+        self.assertEqual(update_resp['Reason'], "invalid request: request type is '{'Update'}' but 'PhysicalResourceId' is not defined")
```

Comment:
Changed in this PR, but the test code has not been modified.
https://github.com/aws/aws-cdk/pull/28057